# Mosquito Data

## Introduction

Mosquito feeding is a combination of processes that are precisely regulated, guided by a hierarchy of stimuli influencing meal acceptance, ingestion, and digestion. The malaria parasite modulates feeding by stimulating mosquito probing and increasing blood meal size. In this study, firstly, we mechanistically dissect the roles of meal temperature, feeding mode and meal composition in shaping the feeding behaviour of Anopheles coluzzii, a major malaria vector, by measuring metrics of the mosquito gut and abdomen. We then explored the role of the Plasmodium-derived intermediate metabolite (E)-4-Hydroxy-3-methyl-but-2-enyl pyrophosphate (HMBPP) in the feeding behaviour of this mosquito by measuring meal size and distribution in the gut. Lastly, we explored the neuromodulatory pathways associated with phagostimulants such as ATP and HMBPP, and how they modulate the transcriptome of the mosquito gut. Our findings reveal that temperature acts as a primary trigger for meal acceptance and phagostimulation, regulating both blood and nectar feeding. We demonstrate that feeding mode dictates nectar acquisition efficiency, influencing how mosquitoes process and sort different meal types. Furthermore, we unpick the complex processes, governed by phagostimulants and other sensory cues, that determine diet destination. We also found that parasite-derived HMBPP and host-derived ATP modulate the expression of behavioural, physiological, and immune response genes in mosquito. By manipulating these feeding mechanisms, we propose an innovative approach: phagostimulant-enhanced feeding-killing traps, designed to exploit mosquito feeding preferences to disrupt malaria transmission. These mechanistic insights not only advance our understanding of mosquito feeding biology but also lay the foundation for precision-targeted, behaviour-driven mosquito control strategies. This research represents a potential paradigm shift in vector control, moving beyond conventional insecticidal approaches toward exploiting the mosquito’s own feeding circuitry.  

## Data characteristics

<img width="649" height="147" alt="Screenshot 2025-08-28 at 15 53 51" src="https://github.com/user-attachments/assets/cba1dac2-db5c-4ad6-b39d-075c7e830031" />
 <pre> ```
   


"Sample Name
(Required)"	"Species Latin Name
(Required)"	Tissue/Organ (Required for Tissue Samples)	Condition	Nanodrop Concentration	Nanodrop (A260/A280)	Nanodrop (A260/A230)	Qubit (ng/µL)	TapeStation RIN	Amount (ng) Qubit
AC_MFS_1H	Anopheles coluzzii	Heads	MFS	65.5	1.81	0.21	10	9.8	150
AC_MFS_2H	Anopheles coluzzii	Heads	MFS	65.3	1.76	0.38	10.7	9.8	160.5
AC_MFS_3H	Anopheles coluzzii	Heads	MFS	57	1.83	0.53	6	9.2	90
AC_MFS_4H	Anopheles coluzzii	Heads	MFS	259.1	1.89	0.37	50.2	9.4	753
AC_MFS_1B	Anopheles coluzzii	Guts	MFS	36.5	1.75	0.1	31.4	10	471
AC_MFS_2B	Anopheles coluzzii	Guts	MFS	79.7	2.1	1.37	114	10	1710
AC_MFS_3B	Anopheles coluzzii	Guts	MFS	162.5	2.11	1.83	>100	10	>1000
AC_MFS_4B	Anopheles coluzzii	Guts	MFS	129.8	2.1	1.8	154	10	2310
AC_MH_1H	Anopheles coluzzii	Heads	HMBPP	193.6	1.85	0.28	43	9.7	645
AC_MH_2H	Anopheles coluzzii	Heads	HMBPP	311	1.87	0.8	50	9.3	750
AC_MH_3H	Anopheles coluzzii	Heads	HMBPP	186.7	1.8	0.61	14.7	9.3	220.5
AC_MH_4H	Anopheles coluzzii	Heads	HMBPP	117.1	1.8	0.28	16	10	240
AC_MH_1B	Anopheles coluzzii	Guts	HMBPP	31.7	2.03	0.06	27.4	10	>1500
AC_MH_2B	Anopheles coluzzii	Guts	HMBPP	318.2	2.07	0.45	>100	9.4	>1500
AC_MH_3B	Anopheles coluzzii	Guts	HMBPP	178.4	2.11	0.3	>100	10	>1500
AC_MH_4B	Anopheles coluzzii	Guts	HMBPP	86	2.07	1.96	87.6	10	1314
AC_MA_1H	Anopheles coluzzii	Heads	ATP	246.3	1.85	0.77	26.8	9.8	402
AC_MA_2H	Anopheles coluzzii	Heads	ATP	161.1	1.85	0.35	31.2	9.5	468
AC_MA_3H	Anopheles coluzzii	Heads	ATP	296.4	1.87	0.82	39.8	8.8	597
AC_MA_4H	Anopheles coluzzii	Heads	ATP	240.6	1.86	0.34	39.6	9.6	594
AC_MA_1B	Anopheles coluzzii	Guts	ATP	84.3	2.09	0.13	95.2	10	1428
AC_MA_2B	Anopheles coluzzii	Guts	ATP	188.9	2.12	0.37	200	10	3000
AC_MA_3B	Anopheles coluzzii	Guts	ATP	192.8	2.08	0.48	162	10	2430
AC_MA_4B	Anopheles coluzzii	Guts	ATP	187.2	2.09	0.48	39.6	10	594
AC_E_1H	Anopheles coluzzii	Heads	Empty	32	1.87	0.09	29	8.5	435
AC_E_2H	Anopheles coluzzii	Heads	Empty	32	1.97	0.09	20	9.9	300
AC_E_3H	Anopheles coluzzii	Heads	Empty	85.9	1.84	0.32	41	10	615
AC_E_4H	Anopheles coluzzii	Heads	Empty	19.3	1.79	0.07	19	9.4	285
AC_E_1B	Anopheles coluzzii	Guts	Empty	343.3	2.09	0.42	37	9.3	555
AC_E_2B	Anopheles coluzzii	Guts	Empty	182.5	2.12	1.22	32	9.8	480
AC_E_3B	Anopheles coluzzii	Guts	Empty	343.3	2.09	0.92	40	9.5	600
AC_E_4B	Anopheles coluzzii	Guts	Empty	182.5	2.12	1.22	35	10	525
      ``` </pre>

## Aims
- Determine transcriptional changes in the guts (midgut and crop) associated with mosquito feeding on minimum stimuli Mix Feeding Solution (MFS)
- Determine transcriptional changes in the gut(midgut and crop) associated with mosquito feeding on MFS and (E)-4-Hydroxy-3-methyl-but-2-enyl pyrophosphate (HMBPP) mix
- Determine transcriptional changes in the gut(midgut and crop) associated with mosquito feeding on MFS and ATP mix

- Determine transcriptional changes in the heads associated with mosquito feeding on minimum stimuli Mix Feeding Solution (MFS)
- Determine transcriptional changes in the heads associated with mosquito feeding on MFS and (E)-4-Hydroxy-3-methyl-but-2-enyl pyrophosphate (HMBPP) mix
- Determine transcriptional changes in the heads associated with mosquito feeding on MFS and ATP mix

Fed at 37°C at the start of the scotophase on a 12:12 light phase, mosquitoes were mated and fed on day 5 after emerging

## Project Pipeline Overview

Raw Read Quality Control
➤ Tool: FastQC
➤ Purpose: Assess the quality of raw sequencing reads.

Read Trimming
➤ Tool: Trimmomatic 
➤ Purpose: Remove adapter sequences and low-quality bases.

Post-trimming Quality Control
➤ Tool: FastQC
➤ Purpose: Confirm improvement in read quality after trimming.

Transcriptome Alignment / Quantification
➤ Tool: Salmon
➤ Purpose: Quasi-mapping reads to the transcriptome and quantifying transcript abundance.

Gene-level Aggregation and Differential Expression Analysis
➤ Tools: tximport → DESeq2
➤ Purpose: Import transcript-level estimates and perform differential gene expression analysis.

## Data Organisation: Directory Structure and Scripts

```mosquito_phagostimulants/
├── FastQC_results/           # Quality reports for raw reads (FastQC output)
├── trimmed_reads/            # Trimmed FASTQ files (from Trimmomatic)
├── trimmed_fastqc_results/   # QC reports for trimmed reads (FastQC)
├── alignments/               # Quantification output (e.g., Salmon results)
├── logs/                     # Log files from trimming, alignment, etc.

git_repos/
├── Reference_genome/         # An. gambiae PEST genome/transcriptome (FASTA, GTF)
├── rnaseq_data/              # Raw data and RNA-seq processing scripts
│   ├── fastqc_array.sh             # Script to run FastQC on raw reads
│   ├── trimmomatic_array.sh        # Script for adapter trimming and quality filtering
│   └──  salmon_alignment.sh             # Script to quantify expression using Salmon 
  ```
  

## Metadata


 <pre> ```
   
   ``` </pre>

## Installation / Requirements

- `FastQC/0.12.1-Java-17`
- `Java/17.0.6`
- `Trimmomatic/0.39-Java-17`
- `R` (v4.2.0)
  - Packages: `DESeq2`, `tximport`, `pheatmap`, `ggplot2

## How to run


https://www.novogene.com/eu-en/wp-content/uploads/sites/4/2020/07/mRNA-demo-report.pdf
